jdaviz/configs/cubeviz/plugins/viewers.py

```diff
@@ -49,6 +51,20 @@ def __init__(self, *args, **kwargs):
         self.add_event_callback(self.on_mouse_or_key_event, events=['mousemove', 'mouseenter',
                                                                     'mouseleave'])
 
+    def _get_cube_skycoord(self, w, x, y):
+        if isinstance(w, WCS):
+            # TODO: Is x always RA and y always DEC?
```

Comment:
I think I am in a WCS rabbit hole. There are several test failures but I cannot tell at a glance if it is because they have always assumed the wrong answer, or this fix is wrong. More investigation is needed.